azureLogIngestion/opentelemetry/nrTime.ts

```diff
@@ -0,0 +1,14 @@
+import { TimedEvent } from "@opentelemetry/tracing/build/src/TimedEvent"
```

Comment:
Not sure this is going to be viable. Any chance we can pull this into a utility or something?